internal/netsync/manager.go

```diff
@@ -474,98 +474,96 @@ func (m *SyncManager) fetchNextHeaders(peer *Peer) {
 	peer.PushGetHeadersMsg(locator, &zeroHash)
 }
 
-// startSync will choose the best peer among the available candidate peers to
-// download/sync the blockchain from.  When syncing is already running, it
-// simply returns.  It also examines the candidates for any which are no longer
-// candidates and removes them as needed.
-func (m *SyncManager) startSync() {
-	// Nothing more to do when already syncing.
-	if m.syncPeer != nil {
-		return
-	}
-
-	chain := m.cfg.Chain
-	best := chain.BestSnapshot()
+// updateSyncPeerState updates the sync peer to the best candidate and removes
+// any sync candidates that are no longer candidates.
```

Comment:
Thanks for the suggestion.  Good point about it being fairly easy to misinterpret remove in this context.  I'll use your wording as is.  It's nice and clear!

I was intentionally being a little bit vague with the "best" wording, because ultimately I'd like to have a fitness function that is used to determine the best candidate and "best" covers that.  However, the comments can always be updated as needed, so I agree it's better to be a little more explicit here.